### PR TITLE
Create 5b4a7cf6-6bcf-11eb-9439-0242ac130002.yml

### PR DESCRIPTION
**Pull request by abhiabhi2306**
This command will display you all the network information you want in one go, this approach is less likely to trigger blueteam monitoring systems.

### CODE REVIEW

File: discovery/5b4a7cf6-6bcf-11eb-9439-0242ac130002.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+id: 5b4a7cf6-6bcf-11eb-9439-0242ac130002 
+metadata:
+  version: 1
+  authors: 
+    - Abhishek S (abhiabhi2306)
+  tags: []
+name: Show whole network information
```
Comment (khyberspache) [inline discovery/5b4a7cf6-6bcf-11eb-9439-0242ac130002.yml:7]: > Dump all network configuration data

File: discovery/5b4a7cf6-6bcf-11eb-9439-0242ac130002.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+description: |
+  This command will display you all the network information you want in one go, this approach is less likely to trigger blueteam monitoring systems.
```
Comment (khyberspache) [inline discovery/5b4a7cf6-6bcf-11eb-9439-0242ac130002.yml:9]: >  This command will display all of the system's network configuration information using an approach that is less likely to trigger blue-team monitoring systems.

**Review by khyberspache — CHANGES_REQUESTED** [review]
Nice one!

**Comment by abhiabhi2306** [comment]
Done, thank you :)

**Comment by w0rk3r** [comment]
@khyberspache this one got the tactic duplicated in the description

**Pull request merged**